virtual_rainforest/core/utility_functions.py

```diff
@@ -0,0 +1,38 @@
+"""The ``core.utility_functions`` module contains functions that are used across the
+Virtual Rainforest, but which don't have a natural home in a specific module. At the
```

Comment:
I've a mild preference for `virtual_rainforest.core.utils` but only really for brevity.

Reply:
Yeah if that is a fairly standard shortening I'm happy to use it